Add nos version 1.1.4

11bDev opened the pull request:
## Summary

This PR adds nos version 1.1.4 which fixes the command alias issue from v1.1.3.

### Changes
- **Fixed command alias**: Users can now type \
os\ instead of the full filename \
os-1.1.4-windows-amd64.exe\
- **Added Commands section**: Includes \Commands: - nos\ in installer manifest
- **Updated checksums**: All checksums updated for v1.1.4 binaries
- **Maintains portable installer**: Continues using portable installer type for standalone executables

### Related
- Fixes command line usability issue reported for v1.1.3
- Release: https://github.com/PlebOne/nos/releases/tag/v1.1.4
- Replaces previous PR #292714 which had merge conflicts

### Validation
✅ \winget validate\ passes successfully
✅ All required manifest files present (installer, locale, version)
✅ Checksums verified against GitHub release assets
✅ Clean branch based on latest upstream master
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/292715)

11bDev committed:
Add nos version 1.1.4

- Fixes command alias issue from 1.1.3
- Users can now type 'nos' instead of full filename
- Includes 'Commands: - nos' in installer manifest
- Updated checksums for v1.1.4 binaries
- Maintains portable installer type for standalone executables

stephengillie commented:
Automatic Validation ended with: 

> Installation failed with exit code -1978335187
> 2025-09-12 20:38:57.747 [FAIL] Installer failed security check. Url: https://github.com/PlebOne/nos/releases/download/v1.1.4/nos-1.1.4-windows-amd64.exe Result: 0x80004005 
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
  | 80004005 | -2147467259 | 2147500037 | E_Fail | Resource missing | 
  | 8A15002D | -1978335187 | 2316632109 | APPINSTALLER_CLI_ERROR_INSTALLER_SECURITY_CHECK_FAILED | Installer failed security check.  Another script should retry this about every 18 hours. <!-- [Policy] Needs-Attention [Policy] Validation-Defender-Error --> | 


(Automated response - build 1111.)

stephengillie commented:
@wingetbot run

stephengillie commented:
Automatic Validation ended with: 

> Installation failed with exit code -1978335187
> 2025-09-13 04:20:58.132 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerSharedLib\SHA256.cpp(144)\WindowsPackageManager.dll!00007FFB7D9C273F: (caller: 00007FFB7DA0AEC4) Exception(1) tid(494) 8A150040 
> 2025-09-13 04:21:03.339 [FAIL] Installer failed security check. Url: https://github.com/PlebOne/nos/releases/download/v1.1.4/nos-1.1.4-windows-amd64.exe Result: 0x80004005 
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
  | 80004005 | -2147467259 | 2147500037 | E_Fail | Resource missing | 
  | 8A15002D | -1978335187 | 2316632109 | APPINSTALLER_CLI_ERROR_INSTALLER_SECURITY_CHECK_FAILED | Installer failed security check.  Another script should retry this about every 18 hours. <!-- [Policy] Needs-Attention [Policy] Validation-Defender-Error --> | 
  | 8A150040 | -1978335168 | 2316632128 | APPINSTALLER_CLI_ERROR_STREAM_READ_FAILURE | Error reading from the stream | 


(Automated response - build 1111.)

11bDev committed:
Update nos 1.1.4 checksums for signed binaries

- AMD64: ad37dd05023b029c6c280a40bf11c7c507d06a6242e2132f2ae3b83c0d610add
- ARM64: b788915ec97566e17a990df77cc62f495af0ecc998b3905d5ba209a81e154ca1

Resolves Windows installation error -1978335187 by providing code-signed executables.
